collection_mutation.cc

```diff
@@ -215,17 +215,19 @@ collection_mutation collection_mutation_view_description::serialize(const abstra
     return serialize_collection_mutation(type, tomb, boost::make_iterator_range(cells.begin(), cells.end()));
 }
 
-collection_mutation
-collection_type_impl::merge(collection_mutation_view a, collection_mutation_view b) const {
-    return a.with_deserialized(*this, [&] (collection_mutation_view_description a_view) {
-    return b.with_deserialized(*this, [&] (collection_mutation_view_description b_view) {
+collection_mutation merge(const abstract_type& type, collection_mutation_view a, collection_mutation_view b) {
+    return a.with_deserialized(type, [&] (collection_mutation_view_description a_view) {
+    return b.with_deserialized(type, [&] (collection_mutation_view_description b_view) {
+
```

Comment:
Please use half/quarter indents here to make the scopes visible.

Reply:
It is fixed in a later commit.